packages/react/src/Card/Card.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import {
+  AnchorHTMLAttributes,
+  ForwardedRef,
+  forwardRef,
+  ForwardRefExoticComponent,
+  HTMLAttributes,
+  PropsWithChildren,
+  RefAttributes,
+} from 'react'
+
+export const CardHeadingGroup = forwardRef(
```

Comment:
Will a `children` prop not provide too much freedom?

I’m inclined to replace it with `heading` and `tagline` props.

Reply:
So something like this?

```
<Card.HeadingGroup key={1}
  heading="Monitor Attracties MRA"
  tagline="Dossier"
>
```

Could work, but we would also have to pass `size` and `level` (and all regular heading props?) for the Heading, and `href` (and all regular link props?) for the Link. Oh, and the props for Paragraph. Together with the regular arguments about inline styling of text, parts of the text being in a different language etc. 

Maybe leave it as is for now?

Reply:
Depends on the amount of variety in the designs. I would expect the tagline to always have the same appearance. The level and size of the heading may well be variable.

What if people add three child elements? They will be surprised to find them all in reverse order.

Combining everything, we could make `tagline` a prop, render its value first, then all children in their own order. Makes it easy to configure the heading. Only if we get cards with a dark background will we need to pass an `inverseColor` prop from `HeadingGroup` to its tagline `Paragraph`, but that’s not very probale. Good idea?

Reply:
> we could make tagline a prop, render its value first

That would cause a wrong DOM order, the tagline comes after the title in the DOM. I think we can make `tagline` a prop, but the order of the DOM elements still needs to be reversed visually.

> What if people add three child elements? They will be surprised to find them all in reverse order.

I don't think this will be a huge issue, if you're adding more elements than just the link in the heading you're probably creating invalid html anyway. An `hgroup` can only contain a heading and one or more paragraphs. If the paragraph is rendered using the `tagline` prop, you can only really add a heading.

I've added a `tagline` prop!